whichone/static/js/ui.js

```diff
@@ -164,23 +149,23 @@ function correctAnswerUI(option) {
     $("#text2a").css("color", "lightgreen");
   }
 
+  transitionCardsOut();
+}
+
+async function transitionCardsOut(primaryDelay = 1000, secondaryDelay = 125) {
   // after 1 second, fade out cards
-  setTimeout(function () {
-    hideDelay = 2000; // delay in ms
-    let opacityDelay = 125;
-    $(".choice").css("opacity", 0);
-    $(".choice").css("cursor", "default");
-    $(".down").css("opacity", 0);
-    $("#stats-popup").css("opacity", 0);
-
-    // after cards have faded, display the stats popup
-    setTimeout(function () {
-      $(".choice").addClass("disabled");
-      $(".time-display").addClass("disabled");
-      $("#stats-popup").removeClass("disabled");
-      $("#stats-popup").css("opacity", 1);
-    }, opacityDelay);
-  }, 1000);
+  await new Promise((resolve) => setTimeout(resolve, primaryDelay));
+  $(".choice").css("opacity", 0);
+  $(".choice").css("cursor", "default");
+  $(".down").css("opacity", 0);
+  $("#stats-popup").css("opacity", 0);
+
+  // after cards have faded, display the stats popup
+  await new Promise((resolve) => setTimeout(resolve, secondaryDelay));
```

Comment:
Marzio's suggestion here might be useful elsewhere. This function looks clean & has no nesting. 

async/await allows us to write code as though it is synchronous.